codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/Decoder.java

```diff
@@ -80,25 +83,30 @@
 
     private final DynamicTable dynamicTable;
     private final HuffmanDecoder huffmanDecoder;
+    private long maxHeaderListSizeGoAway;
     private long maxHeaderListSize;
     private long maxDynamicTableSize;
     private long encoderMaxDynamicTableSize;
     private boolean maxDynamicTableSizeChangeRequired;
 
-    public Decoder() {
```

Comment:
@nmittler - FYI I intended to move all the HPACK code into the `io.netty.handler.codec.http2` package and make it package private so I'm not too concerned about API compatibility here.